pipeline-steps/destroy-service.yaml

```diff
@@ -8,13 +8,11 @@ parameters:
   targetCommand: ''
   builtFrom: ''
   product: ''
+  serviceConnection: ''
```

Comment:
I'm pretty sure destroy-service.yaml is not actually used in practice and could be removed in a separate PR (non-blocking)

Reply:
Removed the destroy yaml file and the associated state in azure pipelines. Deleting code is easy!